packages/components/psammead-media-player/README.md

```diff
@@ -24,11 +24,13 @@ This component to be used at any point on the page, specifically when a media pl
 | `placeholderSrc` | string | No   | `null` | `http://foobar.com/placeholder.png` |
 | `placeholderSrcset` | string | No   | - | `"https://bbc.com/300/cat.jpg 300w, https://bbc.com/450/cat.jpg 450w, https://bbc.com/600/cat.jpg 600w"` |
 | `portrait` | boolean | No   | `false` | `true` |
+| `mediaInfo` | object | Yes   | - | `{ title: 'A vertical video pretending to be a cat title',duration: '0:11',durationSpoken: '0,11',datetime: 'PT11S',type: 'video' }`|
 
 The `src` prop is required, as it tells the component what page it needs to embed.
 The `placeholderSrcset` prop is not required, as it allows image responsiveness and optimization depending on the size of the screen.
 The `title` prop is required for accessibility of the embedded iframe.
 The `portrait` prop is not required, and defaults to `false`. This is to support portrait video content in the future.
+The `mediaInfo` prop is required, as it shows info regarding the media such as title, duration, durationSpoken and type of media either Video or Audio
```

Comment:
Here I think we should add another table documenting the constituent properties of the mediaInfo object and whether they are required etc. following the propType defined at https://github.com/bbc/psammead/blob/0be6fee4b760960535d08b9260c44d4ed1604942/packages/components/psammead-media-player/src/index.jsx#L88

Reply:
done